src/components/HTMLEngineProvider/HTMLRenderers/NextStepEmailRenderer.tsx

```diff
@@ -11,7 +11,14 @@ type NextStepEmailRendererProps = {
 function NextStepEmailRenderer({tnode}: NextStepEmailRendererProps) {
     const styles = useThemeStyles();
 
-    return <Text style={[styles.breakWord, styles.textLabelSupporting, styles.textStrong]}>{tnode.data}</Text>;
+    return (
+        <Text
+            testID="email-with-break-opportunities"
```

Comment:
Can we use `nativeID` instead of `testID`, in the past we had an issue with using `testID` and as far as I know `testID` should be used only for testing purposes. Context: https://github.com/Expensify/App/pull/33265/files#r1437244150

Reply:
@tienifr friendly bump